src/main/java/ch/njol/skript/entity/BoatData.java

```diff
@@ -1,56 +1,44 @@
 package ch.njol.skript.entity;
 
-import java.util.EnumMap;
-import java.util.Locale;
-import java.util.Random;
-
 import ch.njol.skript.Skript;
-import org.bukkit.Material;
-import org.bukkit.entity.Boat;
-import org.bukkit.entity.boat.*;
+import ch.njol.skript.aliases.ItemData;
 import ch.njol.skript.aliases.ItemType;
 import ch.njol.skript.lang.Literal;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
+import org.bukkit.Material;
+import org.bukkit.entity.Boat;
 import org.jetbrains.annotations.Nullable;
 
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Random;
+
+// For <1.21.3 compatability only. 1.21.3+ boats are SimpleEntityDatas
 public class BoatData extends EntityData<Boat> {
 
 	private static final boolean IS_RUNNING_1_21_3 = Skript.isRunningMinecraft(1, 21, 3);
```

Comment:
```suggestion
	private static final boolean IS_RUNNING_1_21_3 = Skript.isRunningMinecraft(1, 21, 2);
```
While theres like a 99.9999999999% chance no one will EVER use 1.21.2, it might be safe to use that here just in case.

Reply:
Oh I thought the changes were made in 1.21.3 for spigot/paper

Reply:
the changes came for 1.21.2, but MC released a new version so damn quickly.
Paper never did a 1.21.2 build, but Spigot did

Reply:
ahhh spigot gotcha